src/Sylius/Bundle/AddressingBundle/Factory/ZoneFactoryInterface.php

```diff
@@ -25,4 +25,11 @@
      * @return ZoneInterface
      */
     public function createTyped($type);
+
+    /**
+     * @param array $members
```

Comment:
Not exactly, the concept is to pass array of members codes, that are used to create zone members and add them to ``Zone``... so probably it should be ``@param array $memberCodes`` :smile: